internal/storage/config.go

```diff
@@ -1,13 +1,13 @@
 package storage
 
-import (
-	"database/sql"
-)
+// CRDBConfig represents CRDB-specific configuration for identity-api.
+type CRDBConfig struct {
+	URI string
+}
```

Comment:
This is somewhere where you could leverage infratographer's x constructs: https://github.com/infratographer/x/blob/main/crdbx/config.go

Reply:
This looks great! I didn't even know we had this available - thanks for pointing it out.

Reply:
`migrate`, `seed-database`, and `serve` all use the `crdbx` constructs and flags now.